source/controllers/include/controllers/impedance/Impedance.hpp

```diff
@@ -64,6 +64,8 @@ class Impedance : public IController<S> {
       damping_; ///< damping matrix of the controller associated to velocity
   std::shared_ptr<state_representation::Parameter<Eigen::MatrixXd>>
       inertia_; ///< inertia matrix of the controller associated to acceleration
+  std::shared_ptr<state_representation::Parameter<bool>>
+      forward_force_; ///< flag to decide of force error should be passed on
```

Comment:
nitpick: I think I would prefer the more explicit and verbose name for the parameter. Also there is a typo in the comment
```suggestion
      feed_forward_force_; ///< flag to decide if force error should be passed on
```